requests/utils.py

```diff
@@ -453,13 +452,27 @@ def get_encoding_from_headers(headers):
     :param headers: dictionary to extract encoding from.
     :rtype: str
     """
+    def parse_header(content_type):
+        #Inner function to parse header
```

Comment:
Why does this need to be an inner function? Why can't we define it at the module level as `_parse_header`?